### PR TITLE
chore: Bump k8s-sidecar to 1.3.0 to align and fix CVEs

### PR DESCRIPTION
**Pull request by jimmidyson**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Bump kommander k8s-sidecar to 1.3.0 to align and fix CVEs

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]

**Comment by hectorj2f** [comment]
@jimmidyson The CI is complaining :/

**Comment by jimmidyson** [comment]
@hectorj2f Passing now.

**Review by hectorj2f — APPROVED** [review]

File: stable/grafana/values.yaml [hunk 1]
```diff
@@ -475,7 +475,7 @@ smtp:
 sidecar:
   image:
     repository: kiwigrid/k8s-sidecar
-    tag: 0.1.151
+    tag: 1.3.0
```
Comment (makkes) [inline stable/grafana/values.yaml:478]: kind of a massive step.

**Review by makkes — APPROVED** [review]

File: stable/grafana/values.yaml (hunk 1 above)
Comment (jimmidyson) [reply stable/grafana/values.yaml:478]: Yeah - I've tried to do my best to validate no breakage by looking at code upstream... and I'm pretty sure we're all good here. Note that it's not as huge as you think: `0.1.151` was last pre-1.0 tag and then only 2 releases post-1.0 so not huge jumps in terms of versions at least.

**Review by armandgrillet — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

**Review by natmegs — APPROVED** [review]

**Comment by jimmidyson** [comment]
Is this the most approved PR in d2iq history? Let's see if we can get to 10 reviews!

**Review by hectorj2f — APPROVED** [review]

**Review by samvantran — APPROVED** [review]
Can I join the approval fun?

**Pull request merged**